version.go

```diff
@@ -1,3 +1,3 @@
 package main
 
-const VERSION = "1.14.2"
+const VERSION = "1.15.0"
```

Comment:
Bumping a minor because there is a new feature (show the error status code) plus the ci updates to keep the build process working.